DOCS/new-license-workflow.md

```diff
@@ -17,23 +24,154 @@ Ask the submitter for any additional info needed, preferably via the Github issu
+
+You're done! Yes, there may be more advanced cases where more work than this is required. If you don't add licenses often, it's unlikely the core team will have assigned you something like that, but if they did or you have any questions at all, please ask! They love to help!
+
+### Test locally
+
+When you send the pull request (below), our [CI/CD](https://en.wikipedia.org/wiki/CI/CD) tool [Travis](https://travis-ci.org/github/spdx/license-list-XML) will run all of the tests to make sure everything is OK.
+
+However, that can take quite a bit of time since it runs the tests for every single license in the list. If you make a mistake, you may be waiting a while to learn about it. Often it's much faster to test things locally first. There are three ways to do this:
+
+#### Use your browser to test the XML file
+
+Most web browsers "speak" XML, which means they're a quick and easy way to tell whether you have valid XML in your file. If you've messed something up (forgotten a closing tag, for instance), the browser will give you an error. If everything is A-OK, the browser will show you your XML file.
+
+To test this, either point your browser at the XML file in your file system using [`file://`](https://en.wikipedia.org/wiki/File_URI_scheme) or drag/drop the XML file onto your browser window.
+
+#### Run all tests on your machine
+
+You can run the same tests Travis will, just on your local machine so it's a little faster than waiting for the Travis return trip.
+
+This requires that you're familiar with the command line and ensuring that script dependencies are all installed. Learning these things is left as an exercise for the reader.
+
+1. Change to the directory where you've cloned the `license-list-XML` repo
+1. Run `make validate-canonical-match`
+1. Go make a cup of tea or something ☕️, because this will take a few minutes to complete
+
+#### Test just the one license you added
+
+If you don't feel like a cup of tea right now, you can run the `make validate-canonical-match` process against a single file instead of the entire corpus of licenses:
+
+1. Change to the directory where you've cloned the `license-list-XML` repo
+1. Run `./test-one-license licenseId`, replacing (of course) the `licenseId` with (naturally) the licenseId of the license in question 
+1. Don't make that cup of tea, since you won't have time
+
+### Send the pull request (PR) for the XML and .txt files
+
+You're nearly done! All that's left is for you to tell the team you're done and the work is ready for review and merging.
+
+To do that, [send a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) (commonly known as a PR) to the project.
+
+If you've never sent a PR before, or if you hit any problems, please let us know! We'll help you with the process so you can successfully contribute to the license list.
```

Comment:
I'll add a note here recommending that the PR message (not the commit messages) include `Fixes #` with the issue number immediately following the hash sign. That way the issue will automatically be closed when the PR is merged.